public/class-admwswp-public.php

```diff
@@ -170,7 +170,7 @@ public static function weblinkWidget($attr)
                     'cart_url' => '',
                     'screen_size' => 'desktop',
                     'pager_type' => 'loadMore',
-                    'show_cart_buttons' => true,
+                    'show_cart_buttons' => false,
```

Comment:
Should we do something like this? Then I assume it would get the value of the global setting unless something is passed in otherwise.

```suggestion
                    'show_cart_buttons' => get_field('showAddToCart', 'options'),
```

Reply:
Yeah that makes sense to me. Might need to handle the case that the global setting null, so maybe just need a "if not null" ternary there.